src/Components/Header.tsx

```diff
@@ -32,143 +32,138 @@ const Header = () => {
   }, []);
   const navigate = useNavigate();
 
-  const handleSubmit = async (e: { preventDefault: () => void }) => {
-    e.preventDefault();
-    logout();
-    navigate("/");
+	const handleSubmit = async () => {
+    // e.preventDefault();
+		logout()
+      navigate("/");
+    
   };
-  console.log("currentUser", currentUser);
-
+  
   return (
     <Box sx={navBox}>
       <Box sx={navBoxInnerMobile}>
-        <Box sx={logo}>
-          <Link to="/">
-            <Avatar alt="Logo" src={logoImg} sx={logoImgStyle} />
+      <Box sx={logo}>
+        <Link to='/'>
+        <Avatar alt="Logo" src={logoImg} sx={logoImgStyle}/>
+        </Link>
+      </Box>
+      <Box>
+        <Box sx={navItems}>
+        <Link to='/newlisting' style={{textDecoration: 'none'}}>
+          <AddCircleOutlineIcon sx={{cursor: 'pointer', width: '30px', height: '30px', fill: '#000'}} />
+        </Link>
+        {currentUser ? 
+				<Box>
+          <Link to='/profile/:id' style={{textDecoration: 'none'}}>
+						{userImg ? 
+							<Box
+							component="img"
+							src={userImg}
+							alt={userName?.charAt(0)}
```

Comment:
I think we agreed that we remove charAt(0)?

Reply:
yeah right!